encrypted-media-respec.html

```diff
@@ -3868,23 +3870,48 @@
             <li><p>If Key System messages contain information derived from a user identifier in a consistent manner, for example such that a portion of the initial Key System message for a specific content item does not change over time and is dependent on the user identifier, then this information could be used by any application to track the device or user over time.</p></li>
           </ul>
 
-          <p>In addition, if a Key System permits keys to be stored and to be re-used between origins, then it may be possible for two origins to collude and track a unique user by recording their ability to access a common key.</p>
+          <p>In addition, if a Key System permits keys or other data to be stored and to be re-used between origins, then it may be possible for two origins to collude and track a unique user by recording their ability to access a common key.</p>
           <p>Finally, if any user interface for user control of Key Systems presents data separately from data in HTTP session cookies [[COOKIES]] or persistent storage, then users are likely to modify site authorization or delete data in one and not the others.
             This would allow sites to use the various features as redundant backup for each other, defeating a user's attempts to protect his or her privacy.
           </p>
+          
+          <p>
+            In addition to the potential for sites and other third-parties to track users, the user agent and/or CDM implementer could build a profile of the user's activities or interests, such as sites using the APIs defined in this specification that the user visits.
+            Such tracking would undermine the privacy protections provided by the rest of the web platform, especially those related to isolation of origins.
```

Comment:
I guess my point was that if you are worried about the User Agent implementor tracking users, you have much bigger problems than EME. The User Agent could simply radio back home with information on every site you visit and every UI element you click, every password you enter etc. This specification does not make that any worse.

I fact some User Agent implementors *do* build up a cloud-stored profile, but it is used only for the benefit of the user, to make preferences, bookmarks etc. available across devices and only done with user consent (I assume).

So, I'd suggest removing "user agent and/or".